etc/org.opencastproject.elasticsearch.index.ElasticsearchIndex.cfg

```diff
@@ -1,10 +1,31 @@
 # Elasticsearch index configuration
 
+# Config that cannot be updated during runtime:
+
 # The base identifier of the Elasticsearch index. Subindices will be named after the following schema:
 # identifier_subindex, e.g. opencast_event. If an index with this name doesn't exist, Opencast will create it.
 # Default: opencast
 #index.identifier=opencast
 
+# Config that can be updated during runtime:
+
 # The index name. Will be used for logging.
 # Default: Elasticsearch
 #index.name=Elasticsearch
+
+# How many times to retry get requests to the Elasticsearch in case of ElasticsearchStatusException.
+# Default: None
```

Comment:
I'll fix these after merge, but IMO defaults should match verbatim with the setting below it.  You and I can parse None == 0, but someone who *isn't* as fluent might not be able to.